### ISSUE TITLE
If you save a blend file without pressing "end configuration" the UI can't be opened without developer tools enabled

### ISSUE THREAD

**Issue by VolkerArt** [Mustard2/MustardUI#157]
Blender 4.1 
Mustard UI 0.31.0.5

Would be useful to be able to forget from time to time and have the UI automatically open itself as if "end configuration" was pressed. As far as I know nothing is permanently lost when ending the configuration anyways.

**Comment by Mustard2** [comment]
Hi @TheCalibrator !

Do you mean that the problem occurs when you save and close the file  without ending the configuration?

**Comment by TheCalibrator** [comment]
Yes, exactly. I found out cause I was wondering where it is in the N panel, after about 10 minutes of looking I went into the addon settings, enabled developer tools and it showed up.

**Comment by Mustard2** [comment]
Good to know, I'll have to check then..

**Comment by Mustard2** [comment]
Tentative fix in #202, where this will be checked as soon as the file is opened.
